packages/ui-react/src/components/Shelf/Shelf.tsx

```diff
@@ -146,7 +146,7 @@ const Shelf = ({
         tilesToShow={tilesToShow}
         wrapWithEmptyTiles={featured && playlist.playlist.length === 1}
         cycleMode={'restart'}
-        showControls={!matchMedia('(hover: none)').matches && !loading}
```

Comment:
`matchMedia('(hover: none)').matches` is always `false` on iOS, causing the chevron buttons to never show. That why I removed it

Reply:
That's because the query detects devices with a mouse. It's expected that this returns `false` on mobile (touch) devices.

Reply:
Maybe I should differentiate in saying showing and rendering. On both IOS and Android they didn't show (because of `opacity: 0` being there originally); But the buttons were accessible on Android, while this condition should  theoretically have prevented that. Quite weird.